usaspending_api/conftest.py

```diff
@@ -192,6 +192,7 @@ def mock_matviews_qs(monkeypatch):
 
 
 def pytest_configure():
+    pass
```

Comment:
Does this actually alter behavior here? `pass` is a noop keyword

Reply:
This was actually to get rid of the whacky behavior to strip all DB connections besides the default when testing, but it's rendered obsolete by Keith's work. I'm going to change it back

Reply:
I understand what the desired behavior was, I was curious if adding the pass actually kept the code execution from the rest of the function